packages/nuxt/.eslintrc.json

```diff
@@ -29,7 +29,13 @@
           "error",
           {
             "buildTargets": ["build-base"],
-            "ignoredDependencies": ["nx", "typescript"]
+            "ignoredDependencies": [
+              "nx",
+              "typescript",
+              "@nx/cypress",
+              "@nx/playwright",
+              "nuxi"
```

Comment:
Ignoring `nuxi` for now, because `eslint` does not like the `nightly` version specifier. Will remove once we go from `nightly` to stable `latest` release.